Musigwa-zz/SpAllocBE
# replace the data structure with a database

---

#### Description of changes made

> A concise description optionally the previous behavior, the solution implemented that reflects the work-done.

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/Musigwa/SpAllocBE.git), and `cd SpAllocBE`
- Check out this branch by `git checkout [the branch for this pr]`
- Run `yarn or npm install` to install the project dependencies
- Steps to set up the local environment
- Steps to configure and run the database service
- Steps to run the project on a local machine
- Steps to test out this PR's functionalities

#### Checklist


#### GitHub issue your changes fix

Resolves #1 

#### Screenshots

|        BEFORE        |       AFTER       |
| :------------------: | :---------------: |
| Before these changes | After the changes |
